seismic-app/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.7'
+
+services:
+
+  sample:
+    container_name: sample
```

Comment:
@tudoramariei I usually leave this out, since the the container name will be autogenerated from the current folder name + service name (when you use compose to manage multiple containers). Oh, this should be `seismic-app` in this context, not `sample`.

Reply:
I'm guessing the `sample` from line 5 should also be changed, shouldn't it?

Reply:
Yes, that's right.